docs/docs/dev_docs/contracts/portals/outbox.md

```diff
@@ -28,14 +28,22 @@ function sendL1Messages(bytes32[] memory _entryKey) external;
 Allows a recipient to consume a message from the `Outbox`.
 
 ```solidity
-function consume(
-  DataStructures.L2ToL1Msg memory _message
-) external returns (bytes32 entryKey);
+  function consume(
+    bytes32 _senderAddress,
+    uint256 _senderVersion,
+    address _recipientAddress,
+    uint256 _recipientChainId,
+    bytes32 _content
+  ) external returns (bytes32 entryKey);
 ```
 
 | Name           | Type        | Description |
 | -------------- | -------     | ----------- |
-| `_message`     | `L2ToL1Msg` | The message to consume |
+| `_senderAddress`     | `bytes32` | The Aztec address of the sender. |
+| `_senderVersion`     | `uint256` | The Aztec version of the sender. |
+| `_recipientAddress`  | `address` | The ethereum address of the recipient. |
```

Comment:
```suggestion
| `_recipientAddress`  | `address` | The L1 address of the recipient. |
```

We should either use "ethereum" everywhere or L1. Earlier in the doc I saw L1 so making this suggestion.